__tests__/app.test.js

```diff
@@ -78,6 +78,28 @@ describe("/api/review/:review_id", () => {
             category: expect.any(String),
             owner: expect.any(String),
             created_at: expect.any(String),
+            comment_count: '0',
+          });
+        });
+    });
+    test("GET - 200: returns the single specified review", () => {
+      return request(app)
+        .get("/api/reviews/2")
+        .expect(200)
+        .then(({ body }) => {
+          expect(body.review.length).toBe(1);
+          expect(body.review[0]).toMatchObject({
```

Comment:
This is something we missed first time, but as this is returning a single review rather than multiple reviews it is better practice to return this as an object rather than an array.